grails-app/controllers/org/broadinstitute/orsp/api/IssueListController.groovy

```diff
@@ -22,7 +23,7 @@ class IssueListController extends AuthenticatedController {
                     id               : it.id,
                     projectKey       : it.projectKey,
                     summary          : IssueUtils.escapeQuote(it.summary),
-                    status           : IssueUtils.escapeQuote(it.getApprovalStatus()),
+                    status           : it.type.equals(IssueType.CONSENT_GROUP.getName()) ? '' : IssueUtils.escapeQuote(it.getApprovalStatus()),
```

Comment:
This is confusing to me to be in a view controller. If Consent Groups shouldn't have a status, then that should be true globally and defined at the `Issue` level, or even better, at the database level.

Reply:
I agree with you. I created a changelog to update approval status value for all consent groups. If you have any other comment, please let me know. Thanks!